assets/js/recipe.js

```diff
@@ -22,6 +22,60 @@ class JSONReader {
     });
   }
 
+  createTooltip(text)
+  {
+    let arr = text.split("/");
+    text = arr[1].replace(/\_/g, " ");
+    text = text.replace('sceptergold', 'build tool');
```

Comment:
oh also, `sceptersteel` is the scan tool

Reply:
I'm not sure if hard coding these name conversions in would be an ideal solution for this. Seeing that this goes off the file names of the images, it seems there isn't really a standard to the naming scheme for them which would allow for this system to work effectively and would end up requiring a number of file names otherwise be hard coded similar to sceptergold here.

I think a solution may be better in something like giving a standard for naming the image files for this solution to be able to work dynamically consistently or possibly adding some more properties into the recipes such as `name` keys potentially.

Reply:
Yes, it's not ideal but the best I can do with the current data. If a name could be added it would be great, but I have no idea how and where the jsons are generated :D

Reply:
All the recipes are stored [here](../tree/master/assets/data/recipes), which are just added by hand whenever needed then added via a `<recipe>` html tag to pages.

All the images are [here](../tree/master/assets/images/icons) if changing the file names would be a better solution but that may require more work, maybe something that @ravenbuilder934 could let us know their opinion on for that part.

Reply:
Yea, I was planning on fixing all the file names eventually. But that would still leave `buildtool` instead of `Build Tool` (correct me if I'm wrong)

Reply:
If you'd want to force `Build Tool`, you can just rename the file to `build_tool` with the current approach.

Reply:
Oh neat, it recognizes underscores. I'll prolly do that later today or tomorrow then

Reply:
there, that's most of 'em done